lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
 
     if (command && command === '/ramal') {
-        const findExtension = text ?
-            extCommand.find.bind(extCommand, text) :
-            extCommand.find.bind(extCommand);
-
-        findExtension(out => {
-            res.send(out);
-        });
+        if (update && update === 'true' && config.extMode === 'SHEET') {
+            extCommand.update(text, out => {
+                res.send(out);
+            });
+        }else{
+            const findExtension = text ?
+                extCommand.find.bind(extCommand, text) :
+                extCommand.find.bind(extCommand);
+
+            findExtension(out => {
```

Comment:
Well noticed. I don't think I wrote this arrow myself but I'll change it. Thanks for pointing out.